lib/openshift/cluster_service_version.rb

```diff
@@ -4,15 +4,12 @@ module BushSlicer
   class ClusterServiceVersion < ProjectResource
     RESOURCE = "clusterserviceversions"
 
-    # @return [BushSlicer::ResultHash] with :success depending on 
+    # @return [BushSlicer::ResultHash] with :success depending on
     # status phase=Succeeded and reason=InstallSucceeded;
-    def ready?(user: user, quiet: false, cached: false)
-    	res = get(user: user, quiet: quiet)
-    	if res[:success]
```

Comment:
this has a purpose. If the `get` call failed then there is no `:parsed` element. In the new version, if `get`  failed, then no method for Nil class will be thrown.